contracts/evoting/evoting.go

```diff
@@ -326,11 +380,10 @@ func (e evotingCommand) shuffleBallots(snap store.Snapshot, step execution.Step)
 	}
 
 	// append the new shuffled ballots and the proof to the lists
-	election.ShuffledBallots = append(election.ShuffledBallots, shuffleBallotsTransaction.ShuffledBallots)
-	election.ShuffledProofs = append(election.ShuffledProofs, shuffleBallotsTransaction.Proof)
+	election.ShuffleInstances = append(election.ShuffleInstances, types.ShuffleInstance{ShuffledBallots: shuffleBallotsTransaction.ShuffledBallots, ShuffleProofs: shuffleBallotsTransaction.Proof, ShufflerPublicKey: shufflerPublicKey})
```

Comment:
way too long 😅
you should first create your `types.ShuffleInstance` and then do the append.